README.adoc

```diff
@@ -156,18 +192,15 @@ The first four properties in the properties section of the project, just define
 and version of Java ([hotspot=java-version file=0]`Java 8`) that Maven uses to compile the application 
 source code.
 
-The next four properties provide a single place to specify values that are used in multiple 
-places throughout the application. For example, the [hotspot=test-server-port file=0]`testServerHttpPort` value is used in 
-both the server configuration (`server.xml`) file and will be used in the test class that 
+The Liberty configuration properties provide a single place to specify values that are used in multiple 
+places throughout the application. For example, the [hotspot=default-http-port file=0]`default.http.port` value is used in
+both the server configuration ([hotspot=httpEndpoint file=2]`server.xml`) file and will be used in the test class that
 you will add (`EndpointIT.java`) in the application itself. This means that you can easily 
 change the port number that the server will run on without having to update the application 
 code in multiple places.
```

Comment:
Change
"The Liberty configuration properties provide a single place to specify values that are used in multiple places throughout the application. For example, the [hotspot=default-http-port file=0]`default.http.port` value is used in both the server configuration ([hotspot=httpEndpoint file=2]`server.xml`) file and will be used in the test class that you will add (`EndpointIT.java`) in the application itself. This means that you can easily change the port number that the server will run on without having to update the application code in multiple places."
To
"Open Liberty configuration properties provide you with a single place to specify values that are used in multiple places throughout the application. For example, the [hotspot=default-http-port file=0]`default.http.port` value is used in both the server configuration ([hotspot=httpEndpoint file=2]`server.xml`) file and will be used in the test class that you will add (`EndpointIT.java`) to the application. Because the `default.http.port` value is specified in the `pom.xml` file, you can easily change the port number that the server runs on without updating the application code in multiple places."